.github/ISSUE_TEMPLATE/UX_Template.yml

```diff
@@ -0,0 +1,58 @@
+name: UX Request
+description: UX Request Form
+labels: [UX/UI Issue, Graphic design]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before opening a UX request, please search for existing issues.
+
+        ---
+
+  - type: textarea
+    id: UX-description
+    attributes:
+      label: UX Description
+      description: Describe the request
+    validations:
+      required: true
+
+  - type: dropdown
+    id: request-type
+    attributes:
+      label: Request type
+      description: "What type of request is this?"
+      options:
+        - "A logo design"
+        - "An icon"
+        - "An infographic/chart"
+        - "A template or design for printed materials"
+        - "Swag design"
+        - "Graphic design not covered by the above"
+
+  - type: dropdown
+    id: user-experience
+    attributes:
+      label: User Experience Request type
+      description: "What type of request is this?"
+      options:
+        - "UX analysis/suggestions for improvement"
+        - "User research"
+        - "User testing"
+        - "Application mockups/designs"
+        - "Website mockups/designs"
+        - "Something else UX-related"
+      
+
+  - type: textarea
+    id: Contacts
+    attributes:
+      label: Engineering Contact
```

Comment:
Although we frequently call ourselves engineers, technically I cannot be one in Canada :) , and maybe a PM might request a logo. Maybe 'Primary contact' instead? (also on next line)

Reply:
changed to Primary Contact